nixos/modules/services/networking/pdns-recursor.nix

```diff
@@ -117,17 +119,53 @@ in {
       '';
     };
 
-    extraConfig = mkOption {
```

Comment:
You should add a `mkRemovedOptionModule` for this

Reply:
Right, I forgot about it.